ci/builders/web_engine.json

```diff
@@ -1,41 +1,66 @@
 {
-    "generators": {
-        "pub_dirs": [
-            "flutter/lib/web_ui/",
-            "flutter/web_sdk/web_engine_tester/"
-        ],
-        "tasks": [
-            {
-                "name": "compile web_tests",
-                "parameters": [
-                    "run",
-                    "compile_tests"
-                ],
-                "scripts": [
-                    "out/host_debug_unopt/dart-sdk/bin/dart",
-                    "flutter/lib/web_ui/dev/felt.dart"
-                ]
+    "builds": [
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_variables": {
+                "download_android_deps": false,
+		"download_emsdk": true
             },
-            {
-                "name": "check licenses",
-                "parameters": [
-                    "check-licenses"
-                ],
-                "scripts": [
-                    "out/host_debug_unopt/dart-sdk/bin/dart",
-                    "flutter/lib/web_ui/dev/felt.dart"
-                ]
+            "gn": [
+                "--build-canvaskit",
+                "--web",
+                "--runtime-mode=release"
+            ],
+            "name": "wasm_release",
+            "ninja": {
+                "config": "wasm_release",
+                "targets": []
             },
-            {
-                "name": "web engine analysis",
-                "parameters": [
-                    "analyze"
+            "generators": {
+                "pub_dirs": [
+                    "flutter/lib/web_ui/",
+                    "flutter/web_sdk/web_engine_tester/"
                 ],
-                "scripts": [
-                    "out/host_debug_unopt/dart-sdk/bin/dart",
-                    "flutter/lib/web_ui/dev/felt.dart"
+                "tasks": [
+                    {
+                        "name": "compile web_tests",
+                        "parameters": [
+                            "run",
+                            "compile_tests"
+                        ],
+                        "scripts": [
+                            "third_party/dart/tools/sdks/dart-sdk/bin/dart",
+                            "flutter/lib/web_ui/dev/felt.dart"
```

Comment:
For all these invocations of felt, it would be better to run the shell script that bootstraps felt itself, which is just `flutter/lib/web_ui/dev/felt`. The shell script will find and use the proper prebuilt dart sdk on its own.